packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1376,22 +1372,22 @@ describe('plugin-meetings', () => {
 
           meeting.setRemoteStream(pc);
           pc.ontrack({track: 'track', transceiver: {mid: '0'}});
-          assert.equal(TriggerProxy.trigger.getCall(0).args[2], 'media:ready');
-          assert.deepEqual(TriggerProxy.trigger.getCall(0).args[3], {type: 'remoteAudio', stream: true});
+          assert.equal(TriggerProxy.trigger.getCall(1).args[2], 'media:ready');
+          assert.deepEqual(TriggerProxy.trigger.getCall(1).args[3], {type: 'remoteAudio', stream: true});
 
           pc.ontrack({track: 'track', transceiver: {mid: '1'}});
-          assert.equal(TriggerProxy.trigger.getCall(1).args[2], 'media:ready');
-          assert.deepEqual(TriggerProxy.trigger.getCall(1).args[3], {type: 'remoteVideo', stream: true});
+          assert.equal(TriggerProxy.trigger.getCall(2).args[2], 'media:ready');
+          assert.deepEqual(TriggerProxy.trigger.getCall(2).args[3], {type: 'remoteVideo', stream: true});
 
           pc.ontrack({transceiver: {mid: '2'}, track: 'track'});
-          assert.equal(TriggerProxy.trigger.getCall(2).args[2], 'media:ready');
-          assert.deepEqual(TriggerProxy.trigger.getCall(2).args[3], {type: 'remoteShare', stream: true});
+          assert.equal(TriggerProxy.trigger.getCall(3).args[2], 'media:ready');
+          assert.deepEqual(TriggerProxy.trigger.getCall(3).args[3], {type: 'remoteShare', stream: true});
         });
       });
       describe('#setUpLocusInfoSelfListener', () => {
         it('listens to the self unadmitted guest event', (done) => {
           meeting.locusInfo.emit({function: 'test', file: 'test'}, 'SELF_UNADMITTED_GUEST', test1);
-          assert.calledOnce(TriggerProxy.trigger);
+          assert.calledTwice(TriggerProxy.trigger);
```

Comment:
What new event are we emitting that is putting these off by one?

Reply:
https://github.com/webex/webex-js-sdk/pull/1847/commits/b23c293455fdd083ee006a4522176375422e81e0#diff-7362c0f286448790746a873a7c764a7bR537

previously we were not adding event listener before parsing locus